includes/class-newspack-popups-inserter.php

```diff
@@ -535,9 +554,41 @@ public static function enqueue_scripts() {
 			return;
 		}
 
+		$is_amp      = self::is_amp();
+		$is_amp_plus = $is_amp && self::is_amp_plus();
+
+		if ( Newspack_Popups_Segmentation::is_admin_user() && ( ! $is_amp || $is_amp_plus ) ) {
+			$admin_script_handle = 'newspack-popups-admin-bar';
+			\wp_register_script(
+				$admin_script_handle,
+				plugins_url( '../dist/admin.js', __FILE__ ),
+				[],
+				filemtime( dirname( NEWSPACK_POPUPS_PLUGIN_FILE ) . '/dist/admin.js' ),
+				true
+			);
+			\wp_register_style(
+				$admin_script_handle,
+				plugins_url( '../dist/admin.css', __FILE__ ),
+				null,
+				filemtime( dirname( NEWSPACK_POPUPS_PLUGIN_FILE ) . '/dist/admin.css' )
+			);
+			\wp_localize_script(
+				$admin_script_handle,
+				'newspack_popups_admin',
+				[
+					'label_visible' => __( 'Prompts Visible', 'newspack-popups' ),
+					'label_hidden'  => __( 'Prompts Hidden', 'newspack-popups' ),
+				]
+			);
```

Comment:
The localization script has to be AMP-Plus'sed too. For this, the `newspack_amp_plus_sanitized` filter can be used, [like here](https://github.com/Automattic/newspack-plugin/blob/c9a56188b103330e451c6870291f6e96cb1f59a5/includes/plugins/class-gravityforms.php#L64).